.github/workflows/on-merge-main.yml

```diff
@@ -38,7 +38,7 @@ jobs:
 
       - name: Install dependencies
         if: ${{ steps.modules-cache.outputs.cache-hit != 'true' }}
-        run: npm install
+        run: npm ci
```

Comment:
This change addresses the score of 6 for "pinned dependencies"

https://securityscorecards.dev/viewer/?uri=github.com/Ed-Fi-Alliance-OSS/ed-fi-alliance-oss.github.io